src/crypto/auth/auth_macros.rs

```diff
@@ -156,3 +160,122 @@ mod bench_m {
 }
 
 ));
+
```

Comment:
Ignore it since it is a fix in sodiumoxide. See https://github.com/sodiumoxide/sodiumoxide/issues/272